css/style.css

```diff
@@ -4,8 +4,251 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    overflow-x: hidden;
 }
 
+header.logo{
+    width: 30px;
+    height: 30px;
+    position: relative;
+    top: 30px;left: 120px;
+}
+
+.navbar{
+  color:rgb(88, 69, 69) 
+  
+   
+}
+.hamburger{
+  display: none;
+}
+
+  ul li a {
+    list-style-type: none;
+    font-size: 15px;
+    position: relative;
+    top: 0px;right: 100px;
+    text-decoration: none;
+    
+}
+  ul li a:hover
+  {
+  color: rgb(235, 106, 31);
+  }
+
+  li {
+    display: inline;
+    padding: 7px;
+    margin: 10px;
+    margin-top: 0;
+    float: right;
+    
+  }
+
+  .photo{
+    background-image: url(/img/first-background.jpg);
+    background-size:cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin: 0;
+    padding: 0;
+    width: 1500px;
+    height: 1000px;
+    align-items: center; 
+    position: relative;
+    top: 0px;left: 30px;
```

Comment:
You shouldn't need these `top` and `left` rules. These are only used to position elements that are not in the normal flow of the page, usually with `position: absolute`. 
Here you should be able to do everything with flexbox. 
Remove all the instances of `top` and `left` in your CSS, and replace fixed widths with percentage widths. For example here, change the width to `width: 100%`